CMakeLists.txt

```
@@ -88,7 +88,7 @@ configure_file (
 include (CTest)
 include (CMakeDependentOption)
 
-option (LBAL_BUILD_DOCS "Build documentation" OFF)
+option (LBAL_BUILD_DOCS "Build documentation" ON)
```

```suggestion
option (LBAL_BUILD_DOCS "Build documentation" OFF)
```
This should be left `OFF` for public releases.